src/core/friendly_errors/fes_core.js

```diff
@@ -144,7 +144,7 @@ if (typeof IS_MINIFIED !== 'undefined') {
         methodParts.length === 1 ? func : methodParts.slice(2).join('/');
 
       //Whenever func having p5.[Class] is encountered, we need to have the error link as mentioned below else different link
-      funcName.substring(0,2) === 'p5.' ?
+      funcName.startsWith(0,2) === 'p5.' ?
```

Comment:
did u test this? i think the first argument of [startsWidth](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) needs to be a string

Reply:
Apologies, fixed and tested it as well now!